tests/gitea-repositories-meta/org42/search-by-path.git/description

```diff
@@ -0,0 +1 @@
+Unnamed repository; edit this file 'description' to name the repository.
```

Comment:
Could you perhaps add a link here to where you got this repo from?
I really don't like not knowing how the repo looked like without major trouble.

Reply:
I've just copied over a fixture repository (BTW copied the repo which was used by the current scenarios) and added a bunch of more files. 

Can you provide some examples of what kind of information do you think would be nice to have on this file ? I've checked the other repos on tests/gitea-repositories-meta and did not found only two repositories that provided a description that was not `Unnamed repository; edit this file 'description' to name the repository.` (in that case, it was `The repository will be used to test third-party renderer in TestExternalMarkupRenderer`). 

Do you think that `This repository will be used to test code search` suffice ?

Reply:
Yes, but that's exactly the problem I can see:
In my experience so far, most of these test repos have been cloned from try.gitea.io or gitea.com at some point.
However, since we don't know from where exactly, we can only guess how the original repo looked like without going through the trouble of converting a non-`.git` folder into one.
Thus, for the vast majority I have no clue what files/commit history they even offer…
But sure, if you don't know either, then it is not needed.

Reply:
@delvh I see your point, and I think it is fair (as a matter of fact, I had the same issue when trying to see the contents of the repositories used by the current test scenarios). What do you think about putting the directory structure of the given repo in its description? I was thinking of something like the snippet below:

```
my-app/
├─ node_modules/
├─ public/
│  ├─ favicon.ico
│  ├─ index.html
│  ├─ robots.txt
├─ src/
│  ├─ index.css
│  ├─ index.js
├─ .gitignore
├─ package.json
├─ README.md
```

Reply:
Sounds good

Reply:
@delvh I've added the directory structure to the description file :)